feat: File Upload Progress (#14878)
sveltejs/kit: .changeset/new-rivers-run.md

---
'@sveltejs/kit': minor
---

> feat: File uploads inside `form` remote functions are now streamed - form data can be accessed before large files finish uploading.

Reviewer: ```suggestion
feat: stream file uploads inside `form` remote functions allowing form data to be accessed before large files finish uploading
```

Author: moving this to #14775
